CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Deprecated
```

Comment:
Can be another PR, but maybe we should have `docs/deprecation.md` in addition to [`docs/experimental.md`](https://github.com/opencontainers/runc/blob/master/docs/experimental.md) ?

Reply:
I don't know. It seems easy enough to search CHANGELOG.md for `eprecat`.